docker-mixin/lib/custom-barchart-grafonnet/custom-barchart.libsonnet

```diff
@@ -0,0 +1,162 @@
+local grafana = import 'github.com/grafana/grafonnet-lib/grafonnet/grafana.libsonnet';
+
+{
+  new(q1, q2, q3)::
```

Comment:
This is very domain specific, which seems intentional given the name `custom-barchart`.

It *might* make sense to allow more queries to be added, and you can use the `addTarget` [pattern which exists in grafonnet](https://github.com/grafana/grafonnet-lib/blob/master/grafonnet/bar_gauge_panel.libsonnet#L39-L45).

Not a necessary change, just a suggestion.

Reply:
Good suggestion! Letting this go for now - Will keep in mind next time.